src/main/java/frc/robot/subsystems/CargoIntake.java

```diff
@@ -68,7 +69,7 @@ public void stopRollers() {
   }
 
   /* TODO: check when limit switches return true */
-  public void spinRollersInWithLimits() {
+  public void spinRollersInWithSensor() {
```

Comment:
Yup this is a more versatile function name, in case we want to use any other sensor including limit switches again.